inference-engine/tests/functional/plugin/cpu/single_layer_tests/roialign.cpp

```diff
@@ -0,0 +1,179 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
Could you please add new SLT to the shared lib?

Reply:
Yes. Done.